core/domain/html_validation_service.py

```diff
@@ -831,27 +831,7 @@ def add_dimensions_to_image_tags(exploration_fs, html_string):
             oppia-noninteractive-image tags.
     """
     soup = bs4.BeautifulSoup(html_string.encode('utf-8'), 'html.parser')
-
-    for image in soup.findAll(name='oppia-noninteractive-image'):
-        if (not image.has_attr('filepath-with-value') or
-                image['filepath-with-value'] == '' or
-                image['filepath-with-value'] == '&quot;&quot;'):
-            image.decompose()
-            continue
-
-        try:
-            filename = json.loads(
-                unescape_html(image['filepath-with-value'].replace('\\"', '')))
-            image['filepath-with-value'] = escape_html(json.dumps(
-                get_filename_with_dimensions(exploration_fs, filename)))
-        except Exception as e:
-            entity_type = exploration_fs.impl.assets_path.split('/')[0]
-            entity_id = exploration_fs.impl.assets_path.split('/')[1]
-            logging.error(
-                '%s %s failed to load image: %s' % (
-                    entity_type, entity_id,
-                    image['filepath-with-value'].encode('utf-8')))
-            raise e
+    _modify_image_filename(exploration_fs, soup)
```

Comment:
_modify_image_filenames

The name is quite vague though -- "modify" can mean anything. Can you change the name to be more explicit about what exactly is being done to the image filenames?

Reply:
Done.